MTV-5019: Preparing the MTV repo for AEM/DITA migration (#897)
kubev2v/forklift-documentation: documentation/doc-Planning_your_migration/assemblies/assembly_migrating-vms-web-console.adoc

[role="_abstract"]

> Use the {project-short} user interface to migrate virtual machines (VMs) from VMware vSphere, {rhv-full} ({rhv-short}), {osp}, Open Virtual Appliances (OVAs) that were created by VMware vSphere, or {virt} clusters. 

Reviewer: ```suggestion
Use the {project-short} user interface to migrate virtual machines (VMs) from {vmw} vSphere, {rhv-full} ({rhv-short}), {osp}, Open Virtual Appliances (OVAs) that were created by {vmw} vSphere, or {virt} clusters.
```

Author: Fixed.

For all migrations, you specify the source provider, the destination provider, and the migration plan. The specific procedures vary per provider.

[IMPORTANT]
====

> You must ensure that all prerequisites are met. For more information, see link:{mtv-plan}assembly_software-requirements-for-migration_mtv[Software requirements for migration].

Reviewer: ```suggestion
You must ensure that all prerequisites are met. 
```

Reviewer: - Moved the links to an Additional resources section.
- Made the same change in my PR https://github.com/kubev2v/forklift-documentation/pull/913
- Ref the `ConceptLink`  Vale rule - https://redhat.atlassian.net/wiki/spaces/OSDOCS/pages/265912624/Content+updates+in+preparation+for+migrating+to+DITA

Reviewer: @RichardHoch Did you not run Vale on the files you updated in the PR? I'm surprised you didn't get the `ConceptLink` error for this assembly file?

Author: @Jenny-Anne I did run vale on the files, but ignored this error because I knew it was covered along with similar errors in other assembly files in another CQA Jira. But you are right, I should have fixed the problem here, instead of leaving it.  I ran the vale check again, just now, to be sure. One product name was flagged in a release note, as always, but on the advice of Marketing, I am leaving it as is until a later date.

> VMware only: You must have the minimal set of link:{mtv-plan}assembly_provider-specific-requirements-for-migration_mtv#ref_vmware-prerequisites_mtv[VMware privileges].

Reviewer: ```suggestion
{vmw} only: You must have the minimal set of {vmw} privileges.
```

> VMware only: link:{mtv-plan}assembly_provider-specific-requirements-for-migration_mtv#proc_creating-vddk-image_mtv[Creating a VMware VDDK image] increases migration speed.

Reviewer: ```suggestion
{vmw} only: Creating a {vmw} Virtual Disk Development Kit (VDDK) image will increase migration speed.
```

====

> 

Reviewer: ```suggestion

[role="_additional-resources"]
== Additional resources
* link:{mtv-plan}assembly_software-requirements-for-migration_mtv[Software requirements for migration]
* link:{mtv-plan}ref_vmware-privileges_mtv[VMware privileges]
* link:{mtv-plan}assembly_provider-specific-requirements-for-migration_mtv#proc_creating-vddk-image_mtv[Creating a VDDK image]

```

Author: @Jenny-Anne  See my comment on https://github.com/kubev2v/forklift-documentation/pull/913 (added it about 30 minutes ago, after making the change you suggested and checking the preview).

Author: ref_vmware-privileges.adoc isn't yet in main, so I used the current reference.
